_includes/generator-form.html

```diff
@@ -0,0 +1,63 @@
+{::options parse_block_html="false" entity_output="as_input" /}
+<div class="tgmpa-generator-form">
+	<form id="generator-form" method="POST">
+		<input type="hidden" name="tgmpa_generate" value="1" />
+
+		<section class="tgmpa-form-primary">
+			<label for="tgmpa-flavor">I'm going to use TGMPA in a:<span class="required">*</span></label>
+			<fieldset>
+				<input type="radio" id="tgmpa-flavor-theme" name="tgmpa-flavor" value="parent-theme" required />
+				<label for="tgmpa-flavor-theme">Theme</label>
+				<br />
+				<input type="radio" id="tgmpa-flavor-child-theme" name="tgmpa-flavor" value="child-theme" required />
+				<label for="tgmpa-flavor-child-theme">Child Theme</label>
+				<br />
+				<input type="radio" id="tgmpa-flavor-plugin" name="tgmpa-flavor" value="plugin" required />
+				<label for="tgmpa-flavor-plugin">Plugin</label>
+			</fieldset>
+		</section>
+
+		<section class="tgmpa-form-secondary">
+			<fieldset>
+			<label for="tgmpa-text-domain">Text Domain:<span class="required">*</span></label>
```

Comment:
I've added the `aria-required` attribute instead, support when having both the HTML5 as well as the WAI attribute is reasonable. http://john.foliot.ca/required-inputs/